build/vsts_build.yaml

```diff
@@ -736,7 +736,7 @@ jobs:
 
   - task: ShellScript@2
     displayName: "Run Tests"
-    continueOnError: "true"
+    continueOnError: ${{ eq(variables['IsOfficialBuild'], 'true') }}
```

Comment:
Wouldn't this prevent you from getting *any* test results?

Reply:
the "upload test results" steps all are configured to run "when successful or failed" (they only don't run when the job is cancelled).

Reply:
ah, good point.